app/models/concerns/speedrun_dot_com_run.rb

```diff
@@ -1,8 +1,6 @@
 require 'active_support/concern'
 
-require 'speedrundotcom'
-
-module SRDCRun
+module SpeedrunDotComRun
```

Comment:
The new autoloader didn't like this old class name, so I updated it to be consistent with all the models that reference speedrun.com stuff.